GliaWidgets/ViewController/Survey/Components/CheckboxView/Survey.Checkbox.swift

```diff
@@ -12,6 +12,18 @@ extension Survey {
             let state: State
             let onSelection: () -> Void
 
+            var accessibility: Accessibility {
```

Comment:
Still has embedded constants. We should pass the from outside in `Accessibility` struct.

Reply:
Haven't done it(

Reply:
Closing this comment in favour of [discussion](https://github.com/salemove/ios-sdk-widgets/pull/280#discussion_r870994059) above.